### ISSUE TITLE
General question on hw7

### ISSUE THREAD

**Issue by BaichenTan** [uc-cfss/Discussion#211]
If we do not do the stretch yourself part for hw7, will our grades be deducted?

**Comment by songcj75** [comment]
I also have the same question, will our code be marked satisfactory instead of excellent if we do not do the stretch part?

**Comment by bensoltoff** [comment]
Not necessarily, it depends on what else you have done in the submission in terms of estimating the models, reporting the results, interpretation, etc.
